### ISSUE TITLE
B018 does not detect module level useless expressions

### ISSUE THREAD

**Issue by graingert** [PyCQA/flake8-bugbear#267]
with the following:

```python
def ham():
    pass


["hello", "world"]
```
when running `pipx run --spec="flake8-bugbear" flake8 fluke8.py --include="B018"`, I expected to see:
```
flake8.py:5:1: B018 Found useless expression. Either assign it to a variable or remove it.
```

**Comment by r-downing** [comment]
ok to close?

**Comment by cooperlees** [comment]
Thanks for noting.
